### PR TITLE
Improve the lightcurve notebook

### PR DESCRIPTION
**Pull request by AtreyeeS**
I think we unnecessarily introduced a lot of misunderstanding by not fitting the model before running the estimator. This is not time consuming, so I adapted the notebook for clarity. Also some other minor modifications to expose more features

### CODE REVIEW

**Review by Astro-Kirsty — CHANGES_REQUESTED** [review]
Thanks @AtreyeeS, just some minor formatting comments

**Comment by AtreyeeS** [comment]
Thanks @Astro-Kirsty ! I adapted the format..

@cgalelli! Yeah I agree. But the rebinning was there already.. Here I also added the slicing to show how to get a flat lc, and then perform some operationon it... We can keep this now and remove it for 2.0

**Review by cgalelli — DISMISSED** [review]
Thanks @AtreyeeS ! 
I don't think there is necessarily the need to remove parts of this tutorial, my comment was more generic in scope, on the organization and distribution of the concepts in the different notebooks.

**Review by bkhelifi — APPROVED** [review]
Nice improvement that shows more features. Thanks a lot

File: examples/tutorials/analysis-time/light_curve_flare.py [hunk 1]
```diff
@@ -301,21 +304,37 @@
...
-# Light curves once obtained can be rebinned.
-# Here, we rebin 4 adjacent bins together, to get 30 min bins
+# Light curves once obtained can be rebinned using the likelihood profiles.
+# Here, we rebin 3 adjacent bins together, to get 30 minute bins.
+#
+# We will first slice `lc_1d` to obtain the lightcurve in the first energy bin
 #
 
-axis_new = get_rebinned_axis(lc_1d, method="fixed-bins", group_size=3, axis_name="time")
+slices = {"energy": slice(0, 1)}
```
Comment (registerrier) [inline examples/tutorials/analysis-time/light_curve_flare.py:320]: This could be shown in the estimators notebook instead, no?

**Review by registerrier — COMMENTED** [review]
Thanks @AtreyeeS . How does this change the CI time? In general fitting with all independent datasets might be lengthy.

File: examples/tutorials/analysis-time/light_curve_flare.py (hunk 1 above)
Comment (QRemy) [reply examples/tutorials/analysis-time/light_curve_flare.py:320]: we can have it repeated here

**Review by QRemy — APPROVED** [review]
Thanks @AtreyeeS, merging now.

**Pull request merged**